### PR TITLE
🎨 Reusable project item component #14

### PR DESCRIPTION
**Pull request by sammuthaiga**
# Description

"Fixes issue #14: Introducing Responsive Project Card Component 🌟

This pull request implements an exciting new feature that brings a dynamic project card to the forefront. Designed to seamlessly adapt to screens of all sizes, this component is geared to be showcased within a carousel. By ensuring optimal responsiveness, the new card component takes user experience to the next level. Say hello to an enhanced and visually appealing way of displaying projects!"
Fixes issue #14

## Type of change
- [x] When rendered the component is pixel-perfect compared to the design above.
- [x] The component is clean and does not have more than 50 lines
- [x] The component SCSS uses CSS variables for colour, font, ... Connected to the main 
      style.scss
- [x] The code is isolated to the libs/elements/cards package unless a justified reason is 
- [x]  specified.
- [x] The model/schema is isolated in the libs/models/ui/cards package unless a justified 
      reason is specified.
- [x] The design is responsive for screens, tablets and phones.
- [x] The image renders fast and has multiple srcsets.
- [x] New feature (non-breaking change which adds functionality)

## Screenshot (optional)
Project-card Item

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [x] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

File: libs/data/sections/src/lib/highlighted-projects.data.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { ProjectItem } from './../../../../models/schema/ui/cards/src/lib/project-item.interface';
```
[the remark this answers is not in the document]
Comment (sammuthaiga) [reply libs/data/sections/src/lib/highlighted-projects.data.ts]: Hi, @LemmyMwaura 
Changes implemented

File: libs/data/sections/src/lib/sections.module.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core';
```
[the remark this answers is not in the document]
Comment (sammuthaiga) [reply libs/data/sections/src/lib/sections.module.ts:1]: I have deleted the file

**Comment by sammuthaiga** [comment]
Hi Lemmy.
I have done all the changes and pushed.
Kindly review.

File: libs/data/sections/tsconfig.lib.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "declaration": true,
+    "declarationMap": true,
+    // "inlineSources": true,
```
Comment (LemmyMwaura) [inline libs/data/sections/tsconfig.lib.json:7]: ??. There's no need for this comment

File: libs/features/pages/home/src/lib/main/home/home-page.component.ts [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
-export class HomePageComponent {}
+export class HomePageComponent {
+  project : ProjectItem = {
+    title:'Mongo Db. Beginner to advanced',
+    description:'MongoDB is a widely-used NoSQL database that offers a flexible and scalable approach to managing data. ',
+    imgSrc:"https://rb.gy/07l7z",
+    tag:'Database'
+  }
+}
+
```
Comment (LemmyMwaura) [inline libs/features/pages/home/src/lib/main/home/home-page.component.ts:16]: No need to push these changes. Any component that needs the project Item will import it.

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

File: libs/features/pages/home/src/lib/main/home/home-page.component.ts (hunk 4 above)
Comment (sammuthaiga) [reply libs/features/pages/home/src/lib/main/home/home-page.component.ts]: Resolved @LemmyMwaura . 
Kindly review

File: libs/data/sections/tsconfig.lib.json (hunk 3 above)
Comment (sammuthaiga) [reply libs/data/sections/tsconfig.lib.json]: Sure.I have uncommented it.

**Pull request closed without merging**